anthos-bm-openstack-terraform/README.md

```diff
@@ -0,0 +1,86 @@
+# Terraform example to create OpenStack VMs for Anthos
+This repository contains an example of what OpenStack resources you need to
+create for setting up Anthos on Bare Metal. The following resources are
+created as part of the Terraform module:
+- External Load Balancer using Octavia LBaaS
+- Private Network for the VMs
+- A router to provide external connectivity and for floating IP addresses
+- 3 Nova VMs: 1 Admin workstation, 1 Control plane and 1 worker node
+- Security Group that allows all ICNM and incoming TCP on port 22 and 443
+  from 0.0.0.0/0
+- Private SSH key that is uploaded to all VMs
+- A cloud-config that sets up SSH access to all VMs from all VMs under
+  the user `abm`
+
+## Getting started
```

Comment:
Will similar instructions live in a DevSite doc? (cloud.google.com/<product>/docs) ?  
Asking because we try not to put tutorial-like instructions in Github.  (ie. Sample code= Github, Words = DevSite) 

I think the first part of this README (explaining the Terraform resources created) is fine, it's just the Getting Started instructions I'm referring to.

Reply:
yes the idea is that similar instructions would live in a DevSite doc. Should I remove the Getting started section when the DevSite doc goes live and then refer to that from this README? I'm happy to remove it now as well even though instructions aren't live yet.

Reply:
Yes, I recommend putting the Getting Started instructions into a devsite doc, then link to that doc from the README of this directory. 
Here's an example- https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/master/hello-app#hello-application-example